app/src/main/java/com/mobileforce/hometeach/models/TutorAllModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.mobileforce.hometeach.models
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.mobileforce.hometeach.adapters.CircleTransform
+import com.mobileforce.hometeach.R
+import com.squareup.picasso.Picasso
+
+data class TutorAllModel(
+
+    val tutorName: String,
+    val tutorImage:String,
+    val description: String,
+    val tutorSubject: String,
+    val perhouramount:Int,
+    var rating: String) {
+
+
+}
+@BindingAdapter("tutor_image")
```

Comment:
you could use the binding Adapter in the Util.kt file. it performs the same function with the one here.